sdk/schemaregistry/azure-schemaregistry/setup.py

```diff
@@ -69,7 +69,7 @@
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
         'msrest>=0.5.0',
-        'azure-core<2.0.0,>=1.19.0'
+        'azure-core<2.0.0,>=1.19.1'
```

Comment:
I think azure core is gonna GA the llc in v1.20 this week, so we could probably just bump the min dep to be 1.20?

Reply:
ok, then I can wait to merge this until tomorrow (I think they said thursday)